commands/svc.cmd

```diff
@@ -18,6 +18,27 @@ DOCKER_COMPOSE_ARGS=()
 DOCKER_COMPOSE_ARGS+=("-f")
 DOCKER_COMPOSE_ARGS+=("${WARDEN_DIR}/docker/docker-compose.yml")
 
+
+## check if dnsmasq is enabled
+if [[ -f "${WARDEN_HOME_DIR}/.env" ]]; then
+    eval "$(grep "^WARDEN_DNSMASQ_ENABLE" "${WARDEN_HOME_DIR}/.env")"
+fi
```

Comment:
This part is not required due to `docker-compose up` extract variables from `.env` file automatically

Reply:
@Den4ik Actually I think it is required because the variable is not needed in `docker-compose up` but in the bash script itself to determine IF the additional docker-compose file is added to the arguments list. I just tried it without that check and `set -x` and it is not defined at that point resulting in dnsmasq starting regardless of the value set in `.env`.

Reply:
Yep, my mistake, you where right